tests/fullstack-test/tpcds_util/tpcds_q18.sql

```diff
@@ -0,0 +1,30 @@
+select i_item_id,
```

Comment:
And add `set tidb_allow_mpp=1; set tidb_enforce_mpp=1; set tidb_isolation_read_engines='tiflash';`